cmd/bbolt/main.go

```diff
@@ -1103,17 +1103,19 @@ func (cmd *benchCommand) Run(args ...string) error {
 	db.NoSync = options.NoSync
 	defer db.Close()
 
+	r := rand.New(rand.NewSource(42))
```

Comment:
Question: why use 42 instead of `rand.New(rand.NewSource(time.Now().UnixNano()))`?

Reply:
In the work I'm doing I needed to be able to try out different changes and easily compare runs. Having the seed be the same each time lets me do that. I'm not wedded to the concept, though.

Reply:
Would this be considered a breaking change, given that this random seed is also used for the random write benchmarks?

An option would be to add a new argument to override a default seed (`time.Now().UnixNano()`).

Reply:
I don't think we should use fixed number. Please use random number, like time.Now().UnixNano here.